code_annotations/config_and_tools/config/setting_annotations.yaml

```diff
@@ -0,0 +1,12 @@
+source_path: ./
+report_path: reports
+safelist_path: .annotation_safe_list.yml
+annotations:
+  setting:
+    - ".. setting_name:":
+    - ".. setting_default:":
+    - ".. setting_description:":
+    - ".. setting_warning:":
```

Comment:
After speaking with @feanil, we wondering if `config` is a better term than `setting`?  Note, neither is as clear as when we prefix with the word `non-toggle`, but that probably needs to be reserved for documentation.

Just like Django Settings, ConfigModels are sometimes used for toggles and sometimes used for non-toggles.  We might need a `.. config_implementation:` with `DjangoSetting` and `ConfigModel` options.  I'm not clear one how and if Site Config should be documented, but that could also come into play here.

what do you think of using the term `config` for (non-toggles) and `toggle` (for toggles).  A Django Setting might be used for `config` or for a `toggle`.

Maybe add a brief ADR with what we decide for naming around this, and then update this PR accordingly (if necessary).  Not sure where you and @feanil will land.

Reply:
I think "setting" is better suited than "config". What does "config" mean? It's the abbreviation for "configuration". As such, a configuration is a set of multiple configuration entries -- or "settings". I understand the need to implement a `setting_implementation` annotation, though; we might have to amend the model in the future.

Reply:
I'm fine with this for now, I think we'll see what parts are actually confusing for people and then we can fix forward.